### PR TITLE
docs: update moderator onboarding guide

### PR DESCRIPTION
**Pull request by Om-Aditya-Jain**
Updated moderator-onboarding-guide.md by improving the language of the doc and removed grammatical errors

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by Om-Aditya-Jain — COMMENTED** [review]
@ojeytonwilliams @naomi-lgbt PTAL, Thanks!

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: docs/moderator-onboarding-guide.md [hunk 1]
```diff
@@ -20,33 +20,33 @@ To get familiar with the new tools and powers, you can combine one or more of th
...
 
-Commonly, spam flags can also be raised by members or moderators. Common duplicitous behavior would involve opening an account, making a post that seems harmless, then editing that post later on to add a link to an external site for the purpose of advertising it. In this case, usually the member account is fairly new and has only made this one post thus far, which indicates that the account was opened for the sole purpose of spamming the community. The decision should be made to delete the account after the first offence in this case. The same applies for accounts whose first post is deemed to be spam.
+Commonly, spam flags can also be raised by members or moderators. Common duplicitous behavior would involve opening an account, making a post that seems harmless, then editing that post, later on, to add a link to an external site to advertise it. In this case, the member account is usually fairly new and has only made this one post thus far, which indicates that the account was opened solely for spamming the community. The decision should be made to delete the account after the first offense in this case. The same applies to accounts whose first post is deemed to be spam.
```
Comment (jeremylt) [inline docs/moderator-onboarding-guide.md:31]:
```suggestion
Commonly, spam flags can also be raised by members or moderators. Common duplicitous behavior would involve opening an account, making a post that seems harmless, then editing that post later on to add a link to an external site to advertise it. In this case, the member account is usually fairly new and has only made this one post thus far, which indicates that the account was opened solely for spamming the community. The decision should be made to delete the account after the first offense in this case. The same applies to accounts whose first post is deemed to be spam.
```
In general, the, answer, is, rarely, to, add, more, commas

**Review by Om-Aditya-Jain — COMMENTED** [review]
@ojeytonwilliams @naomi-lgbt updated suggested changes PTAL, Thanks!

File: docs/moderator-onboarding-guide.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This guide will help new moderators get up and running with the processes and pr
 
 ### First Steps
 
-The first thing you may notice after being given moderator status on the forum is that your interface will look somewhat different, with new admin tools to explore and access to the Mod-Team subforum.
+The first thing you may notice after being given moderator status on the forum is that your interface will look somewhat different, with new admin tools to explore and access the Mod-Team subforum.
```
[the remark this answers is not in the document]
Comment (Om-Aditya-Jain) [reply docs/moderator-onboarding-guide.md]: Done.

File: docs/moderator-onboarding-guide.md (hunk 1 above)
Comment (Om-Aditya-Jain) [reply docs/moderator-onboarding-guide.md]: Done.

**Review by huyenltnguyen — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**